core/workspace_svg.js

```diff
@@ -89,6 +89,11 @@ Blockly.WorkspaceSvg = function(options, opt_blockDragSurface, opt_wsDragSurface
    * @private
    */
   this.highlightedBlocks_ = [];
+
+  this.registerToolboxCategoryCallback(Blockly.VARIABLE_CATEGORY_NAME,
+      Blockly.Variables.flyoutCategory);
+  this.registerToolboxCategoryCallback(Blockly.PROCEDURE_CATEGORY_NAME,
+      Blockly.Procedures.flyoutCategory);
```

Comment:
I was working on the assumption that toolboxes are only in rendered workspaces.  This may not be true in light of accessible blockly.  I'll check in with Cory and Sean.

This assumption was the same as the one for the buttons.